### ISSUE TITLE
Array of arrays?

### ISSUE THREAD

**Issue by SandorAlbert** [benmaier/binpacking#14]
Hi, I have an array of arrays where I would like to minimize the number of bins such that the sum does not exceed e.g. [200, 200, 200]. Is this possible using your package? Thank you.

**Comment by benmaier** [comment]
sorry for the late reply! I'm not sure I quite understand. You could map your arrays to their sum and then binpack those? Clarifying your question with a specific example would help!

**Comment by SandorAlbert** [comment]
@benmaier 

Hi, sorry for not making clear what I need to do. I basically have an array of arrays instead of an array of just integers/floats. E.g. [[2, 2, 1], [1, 1, 1], [5, 5, 5]], whereas one bin has size [5, 5, 5]. The optimal solution in this case would be {bin1: [[2, 2, 1], [1, 1, 1]], bin2: [5, 5, 5]]. Is your package capable of working with such arrays? 

I managed to do it using lots of if/else statements and super complicated using the best-fit algorithm. I declared a best fit bin as the bin where the least space in total remains, calculated as the mean of the differences with respect to each index.

Thanks!
